alfresco-webscripts-client/alfresco-webscripts-spi/src/main/java/eu/xenit/alfresco/webscripts/client/spi/model/slingshot/Metadata.java

```diff
@@ -45,13 +48,19 @@
     @AllArgsConstructor
     public static class ValueContainer {
         String dataType;
-        String value;
+        String value; // TODO Serializable instead maybe?
+        @JsonSetter("isContent")
```

Comment:
Why do we need this :thinking: ?

Reply:
It won't recognize the field for deserialization without it. 
I guess it had something to do with the lombok @Data

I'll try to reproduce